### PR TITLE
Exclude metadata text tracks from the transcribe

### LINKED ISSUE #1: Metadata text tracks should be excluded from the transcribe element
Text tracks on VideoJS can have a kind 'metadata'. This mode means that the track will update as the video is playing, but it won't be visible to the viewer. This is most useful for tracks where kind="metadata" . A common use case for metadata text tracks is to use them to trigger behaviors when their cues become active.

We don't want these to show in the transcript, as they will confuse users.

### PR DESCRIPTION
**Pull request by amaitu**
Fixes: https://github.com/7Ds7/videojs-vjstranscribe/issues/1

### CODE REVIEW

**Comment by 7Ds7** [comment]
Thank you, this looks good.
Never used the metadata tracks so i was totally unaware.

I will make a few tests before merging and prepare a new release.

File: src/videojs-vjstranscribe.js [hunk 1]
```diff
@@ -91,25 +91,32 @@ class VjsTranscribe extends Plugin {
...
+    return tracks.filter((track) => {
+      return track.kind !== 'metadata';
+    })
+  }
+
   getActiveTrack() {
     let i, track;
-    for (i = 0; i < this.player.textTracks().tracks_.length; i++) {
-      track = this.player.textTracks().tracks_[i];
+    for (i = 0; i < this.getTextTracks.length; i++) {
```
Comment (7Ds7) [inline src/videojs-vjstranscribe.js]: This should be `this.getTextTracks().length`

**Review by 7Ds7 — CHANGES_REQUESTED** [review]
The transcribe-selector should also use the same `getTextTracks()` method or else it would be out of sync with the player.

Please let me know if you are willing to correct that or if i should proceed with the changes.

File: src/videojs-vjstranscribe.js (hunk 1 above)
Comment (amaitu) [reply src/videojs-vjstranscribe.js]: ah yeah nice, thanks

**Comment by amaitu** [comment]
> The transcribe-selector should also use the same `getTextTracks()` method or else it would be out of sync with the player.
> 
> Please let me know if you are willing to correct that or if i should proceed with the changes.

Thanks if you are able to do it that would be good, but if not I may have time later in the week or next week.

**Comment by amaitu** [comment]
> Thank you, this looks good. Never used the metadata tracks so i was totally unaware.
> 
> I will make a few tests before merging and prepare a new release.

Thanks for making this repo - I am hoping to use it in production. If I find any other bugs I will put them in issues and make a PR if possible.

**Pull request merged**